app/utils/pool-utils.ts

```diff
@@ -14,10 +14,12 @@ const iconMapping = {
     "SLES": Icons.openSUSE,
     "SLES-HPC": Icons.openSUSE,
     "standard-data-science-vm": Icons.windows,
+    "rendering-windows2016": Icons.windows,
```

Comment:
Good point about that. I kind of missed the way these VM's were displayed and have only now seen the Data Science VM's tab. Makes perfect sense to have them in another tab. Will do that in this PR.